apiserver/src/main/java/com/okbut/api/checkout/domain/Checkout.java

```diff
@@ -0,0 +1,38 @@
+package com.okbut.api.checkout.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name="CHECK_OUT")
+public class Checkout {
```

Comment:
테이블명은 Check와 Out이 나뉘는데 클래스명은 한 단어로 보입니다.

```suggestion
@Table(name="CHECK_OUT")
public class CheckOut {
```

Reply:
통일성을 위해서 전부 소문자로 한건데,, entity만 일단 바꿀게요